scripts/playwright/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "playwright",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "TRACE=true npx playwright test --workers=4",
+    "test:repeat": "TRACE=true npx playwright test --workers=4 --repeat-each=10",
+    "quick-test": "TRACE=true PW_QUICK_TEST=1 npx playwright  test --workers=4",
+    "test-debug": "./startPlayWrightServer.sh; PW_TEST_REUSE_CONTEXT=1 PW_TEST_CONNECT_WS_ENDPOINT=ws://127.0.0.1:31000/ PWDEBUG=console npx playwright test -c playwright.config.ts --headed --project=chromium --retries 0 --timeout 0 --workers 1 --max-failures=1",
+    "test-debug-quick-sqlite": "./startPlayWrightServer.sh; PW_QUICK_TEST=1 PW_TEST_REUSE_CONTEXT=1 PW_TEST_CONNECT_WS_ENDPOINT=ws://127.0.0.1:31000/ PWDEBUG=console npx playwright test -c playwright.config.ts --headed --project=chromium --retries 0 --timeout 5 --workers 1 --max-failures=1",
+    "ci-test-mysql": "E2E_DB_TYPE=mysql npx playwright test --workers=2"
+  },
```

Comment:
Please follow the format of the current packages in develop branch. You may checkout frontend one.